xbmc/PasswordManager.cpp

```diff
@@ -105,6 +105,16 @@ void CPasswordManager::SaveAuthenticatedURL(const CURL &url, bool saveToProfile)
   m_temporaryCache[GetServerLookup(path)] = authenticatedPath;
 }
 
+bool CPasswordManager::IsURLSupported(const CURL &url)
+{
+  if ( url.IsProtocol("smb")
```

Comment:
style suggestion: return url.IsProtocol("smb") || ..

Reply:
current style allows see a result in debugging.